src/interfaces/store-interfaces/state-interfaces/businessInformation-interface.ts

```diff
@@ -1,6 +1,7 @@
+/** Data object used internally only (not to/from API). */
 export interface BusinessInformationIF {
   legalType: string
   identifier: string
   legalName?: string
-  foundingDate?: Date
+  foundingDateTime?: string
```

Comment:
Fixed mismatch between Date and String -- they were used interchangeably in some places, and while JS lets you do that, it's harder to figure out why things sometimes don't work.